src/hooks/helper/price.ts

```diff
@@ -1,12 +1,25 @@
+import { wrappedTokenMap } from './../../modules/token/index';
 import axios from 'axios';
 
 const getUsdPriceViaDia = async (symbol: string): Promise<number> => {
   try {
     // Ref: https://docs.diadata.org/documentation/api-1/api-endpoints#quotation
-    const url = `https://api.diadata.org/v1/quotation/${symbol}`;
+    const url = `https://api.diadata.org/v1/quotation/${symbol.toUpperCase()}`;
```

Comment:
I suggest that we move this out to the Token API too

Reply:
sure, @bobo-k2 could you take a look and provide the endpoint? 🙏

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/460/commits/36b8906d3e3e792dd9f073c8df20a4246dc12a96) commit.
Thank you, @bobo-k2.